lib/fungus_toast/games/grid.ex

```diff
@@ -0,0 +1,32 @@
+defmodule FungusToast.Games.Grid do
+    alias FungusToast.Games.GridCell
+
+def create_starting_grid(number_of_rows, number_of_columns, player_ids) do
+    initial_grid_cells_map = map_of_player_id_to_random_grid_index(number_of_rows, number_of_columns, player_ids)
+
+    Enum.reduce(initial_grid_cells_map, %{}, fn {player_id, position}, map ->
+      Map.put(map, position, %GridCell{
+        index: position,
+        player_id: player_id,
+        live: true,
+        empty: false
+      })
+    end)
+  end
+
+  defp map_of_player_id_to_random_grid_index(number_of_rows, number_of_columns, player_ids) do
```

Comment:
I personally like doing these in two ways: one with a list of IDs, one with a list of Players that can be broken down into a list of IDs or the IDs can be turned into a list of players. Thoughts? If you look at most of my helpers, I try to do that with the struct or the ID just so we have more flexibility in how we call the functions